.moai/specs/SPEC-V3R3-BRAIN-001/spec-compact.md

```
@@ -0,0 +1,125 @@
+
+1. Discovery — Socratic interview (clarity score ≤ 3 triggers up to 5 rounds)
+2. Diverge — 5-15 angle variations (reuses moai-foundation-thinking)
+3. Research — parallel WebSearch + Context7 → research.md
+4. Converge — Lean Canvas → ideation.md
+5. Critical Evaluation — First Principles + Critical Eval (appended to ideation.md)
+6. Proposal — proposal.md with "SPEC Decomposition Candidates" section
+7. Handoff — claude-design-handoff/ (5 files: prompt/context/references/acceptance/checklist)
+
+## Output Layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)